internal/distro/fedora/distro.go

```diff
@@ -1025,11 +1025,44 @@ func newDistro(version int) distro.Distro {
 		imageInstallerImgType,
 	)
 	aarch64.addImageTypes(
-		&platform.Aarch64{
+		&platform.Aarch64_IoT{
 			BasePlatform: platform.BasePlatform{
 				ImageFormat: platform.FORMAT_RAW,
 			},
 			UEFIVendor: "fedora",
+			BootFiles: [][2]string{
```

Comment:
I think you should be able to remove `[2]string` from all of these lines, which would make the code cleaner.

Reply:
+1, there should be no need to specify `[2]string` on each line

Reply:
Done :)